tests/ifs/ifs.c

```diff
@@ -134,7 +134,7 @@ static int scan_common_init(struct test *test)
         int enforce_run = get_testspecific_knob_value_int(test, "enforce_run", -1);
         if (memcmp(status_buf, "fail", strlen("fail")) == 0 && enforce_run != 1 )
         {
-            log_skip(ResourceIssueSkipCategory, "Previous run failure found! This test will skip until enforced adding flag: "
+            log_skip(TestResourceIssueSkipCategory, "Previous run failure found! This test will skip until enforced adding flag: "
```

Comment:
I think @AthenasJimenez wants us to change the message to

`The previous execution encountered an error, and this test will not proceed unless it is explicitly enforced with knob enforce_run=1.`